Recast/PodcastDetailViewController.swift

```diff
@@ -0,0 +1,200 @@
+//
+//  PodcastDetailViewController.swift
+//  Recast
+//
+//  Created by Jack Thompson on 10/16/18.
+//  Copyright © 2018 Cornell AppDev. All rights reserved.
+//
+
+import UIKit
+import SnapKit
+import Kingfisher
+
+class PodcastDetailViewController: UIViewController, EpisodeFilterDelegate {
+
+    // MARK: - Variables
+    var partialPodcast: PartialPodcast!
```

Comment:
do we want to make an init with a `PartialPodcast` input? bc we wouldn't have a podcast detail view controller without a partial podcast